code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1080,6 +1080,17 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	if(attacker_style?.grab_act(user,target) == MARTIAL_ATTACK_SUCCESS)
 		return TRUE
 	else
+		if (target.body_position == LYING_DOWN && (user.zone_selected == BODY_ZONE_L_LEG || user.zone_selected == BODY_ZONE_R_LEG) && target.shoes && istype(user.get_active_held_item(), /obj/item/hand_item/stealer))
```

Comment:
You're overcomplicating this. Just check when the shoe steal hand attacks someone. No reason to care about intent or zone.

Reply:
Why is this on the main grab proc to begin with? Couldn't you just put it on the /obj/item/hand_item/stealer and register for COMSIG_SOMETHING_GRABBED? I'm not really a fan of putting this kind of specialist behaviour on the main grab proc

Reply:
yea